frontend/src/components/grantaccess/form/UserNameInput.vue

```diff
@@ -72,7 +72,7 @@ const verifyUserId = async () => {
         }
     } finally {
         if (verifiedUserIdentity.value?.found) {
-            emit('setVerifyResult', true, verifiedUserIdentity.value.guid);
+            emit('setVerifyResult', true, verifiedUserIdentity.value.guid, verifiedUserIdentity.value.email);
```

Comment:
emit searched user 'email'. This is then used for displaying "failed to send email to" error message based on mockup.